homeassistant/components/lifx/coordinator.py

```diff
@@ -138,6 +138,17 @@ async def async_update_color_zones(self) -> None:
             if zone == top - 1:
                 zone -= 1
 
+    def async_get_hev_cycle_state(self) -> bool | None:
+        """Return the current HEV cycle state."""
+        if self.device.hev_cycle is None:
```

Comment:
All Light objects created by aiolifx get this set to None, but it will also cover the case if a HEV bulb hasn't managed to have its HEV state retrieved yet.

Reply:
I still need to write a test case for this.